test/powershell/Modules/Microsoft.PowerShell.Utility/typedata.tests.ps1

```diff
@@ -1,19 +1,23 @@
 ﻿Describe "TestData cmdlets" -Tags "CI" {
     Context "Get-TypeData" {
         It "System.Array" {
-            (Get-TypeData System.Array).TypeName | Should Be System.Array
+            Get-TypeData System.Array | Should Be $null
```

Comment:
@daxian-dbw You could just [remove](https://github.com/PowerShell/PowerShell/compare/80c03762597de92572403bb090ef7f01984a6cf7...0a1c96d4917ca09cd2e4a5ecd9b5b0c9591521c5) one last commit.